components/mention/demo/form.ts

```diff
@@ -44,27 +55,25 @@ import { NzMentionComponent } from 'ng-zorro-antd/mention';
 })
 export class NzDemoMentionFormComponent {
   suggestions = ['afc163', 'benjycui', 'yiminghe', 'RaoHai', '中文', 'にほんご', 'ParsaArvaneh'];
-  validateForm: FormGroup<{ mention: FormControl<string | null> }>;
+  validateForm = this.fb.group({
+    mention: ['@afc163 ', [Validators.required, this.mentionValidator]]
+  });
   @ViewChild('mentions', { static: true }) mentionChild!: NzMentionComponent;
 
   get mention(): FormControl<string | null> {
     return this.validateForm.controls.mention;
   }
 
-  constructor(private fb: FormBuilder) {
-    this.validateForm = this.fb.group({
-      mention: ['@afc163 ', [Validators.required, this.mentionValidator]]
-    });
-  }
+  constructor(private fb: FormBuilder) {}
 
-  mentionValidator: ValidatorFn = (control: AbstractControl) => {
+  mentionValidator(control: AbstractControl): ValidationErrors {
```

Comment:
I think it really should be a function, not a class method, WDYT?

Reply:
Agree with you. fixed